Gruntfile.js

```diff
@@ -135,6 +135,25 @@ module.exports = function(grunt) {
     jasmine_node: {
       integration: ['spec/integration/']
     },
+    integration: {
+      all: {
+        options: {
+          template: 'node_modules/freedom-for-chrome/spec/helper/',
+          spec: ['spec/chrome_integration/gtalk.spec.js'],
+          helper: [
+            // "include: true" is needed for dependencies that are used by
+            // the jasmine tests in the core environment.
+            {path: 'dist/social.google.json', include: false},
```

Comment:
What notation are you thinking?  I don't think we want to include everything in dist/, for example both google-auth.js and facebook-auth.js will clobber each other.